src/actions/lesson.jsx

```diff
@@ -1,6 +1,4 @@
-import {editor} from "~/utils/socket"
-import { useContext } from "react"
-import LessonContext from "~/Contexts/Lesson"
+import {editorWS} from "~/utils/socket"
```

Comment:
Der Name lesson.jsx könnte auch direkter in sections umbenannt werden.

Reply:
Da kommt not ein wenig mehr rein, aber man kann überlegen ob man das Ausplittet